app/src/main/res/values/strings.xml

```diff
@@ -221,87 +221,129 @@
-         changes the app and the language of the speech-recognition-based search to 'Korean'. -->
+    changes the app and the language of the speech-recognition-based search to 'Korean'.
+    The string is also used in the keyboard space bar to show the current keyboard language and in
+    the keyboard languages panel list. -->
     <string name="settings_language_korean">Korean</string>
 
     <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
-         changes the app and the language of the speech-recognition-based search to 'Italian'. -->
+    changes the app and the language of the speech-recognition-based search to 'Italian'.
+    The string is also used in the keyboard space bar to show the current keyboard language and in
+    the keyboard languages panel list. -->
     <string name="settings_language_italian">Italian</string>
 
     <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
-     changes the app and the language of the speech-recognition-based search to 'Russian'. -->
+    changes the app and the language of the speech-recognition-based search to 'Russian'.
+    The string is also used in the keyboard space bar to show the current keyboard language and in
+    the keyboard languages panel list. -->
     <string name="settings_language_russian">Russian</string>
 
     <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
-     changes the app and the language of the speech-recognition-based search to 'Polish'. -->
+    changes the app and the language of the speech-recognition-based search to 'Polish'.
+    The string is also used in the keyboard space bar to show the current keyboard language and in
+    the keyboard languages panel list. -->
     <string name="settings_language_polish">Polish</string>
 
     <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
-     changes the app and the language of the speech-recognition-based search to 'Danish'. -->
+    changes the app and the language of the speech-recognition-based search to 'Danish'.
+    The string is also used in the keyboard space bar to show the current keyboard language and in
+    the keyboard languages panel list. -->
     <string name="settings_language_danish">Danish</string>
 
     <!-- This string is used to label a radio button in the settings language dialog that, when pressed,
-     changes the app and the language of the speech-recognition-based search to 'Norwegian'. -->
+    changes the app and the language of the speech-recognition-based search to 'Norwegian'.
+    The string is also used in the keyboard space bar to show the current keyboard language and in
+    the keyboard languages panel list. -->
     <string name="settings_language_norwegian">Norwegian</string>
```

Comment:
So I was wondering if we need this string for "Norwegian"? We already have both Norwegian Bokmal and Norwegian Nynorsk below, which are the usual Norwegian locales (and should suffice, I believe). 
Unless it's necessary on your side for some reason to add "Norwegian" as well? Honestly no idea :)

Reply:
We are using the "Norwegian" string for the keyboard spacebar and the keyboard language list as there are not specific keyboard layouts for Bokmal and Nynorsk. Actually this happens also with Spanish/Spanish(Spain) and English(US)/English(UK) so I've updated those too.

I've also updated all the comments to reflect that we are using the language strings both in the Display and the Voice search languages settings panels.